src/aics-image-viewer/components/shared/SliderRow/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { NouisliderProps } from "nouislider-react";
+import SmarterSlider from "../SmarterSlider";
+
+import "./styles.css";
+
+type SliderRowProps = {
+  label: React.ReactNode;
+  start?: number | number[];
+  step?: number;
+  formatInteger?: boolean;
+  max?: number;
+  onUpdate?: NouisliderProps["onUpdate"];
+  onChange?: NouisliderProps["onChange"];
+  hideSlider?: boolean;
+};
+
+const INTEGER_FORMATTER = { to: Math.round, from: Number };
+
+/** A component to ensure a single unified style across the many labeled slider rows in the control panel */
+const SliderRow: React.FC<SliderRowProps> = (props) => (
+  <div className="viewer-control-row">
+    <div className="control-name">{props.label}</div>
+    <div className="control">
+      {props.start === undefined
+        ? props.children
```

Comment:
Is there ever a situation where `props.start` is undefined?

Reply:
When this component has `children`, they replace the slider that the component otherwise renders itself. I did this so I could use the same component for the "interpolate" control:
![image](https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/53030214/3d655ca8-c8f4-4aa8-8b44-41ced95ecee4)
Without a slider, `start` doesn't make much sense, so it's optional. I null-check it here rather than `children` for type-checking reasons.

Reply:
This comment makes it seem like checkbox is just some kind of special-case exception to sliders.  It would be interesting to think of these rows as generic "input rows" where there's a label and an input control that can be anything.